### ISSUE TITLE
How do you use teachers_pet?

### ISSUE THREAD

**Issue by tarebyte** [education/teachers_pet#96]
Hi Everyone,

I'm trying to get a more in depth understanding of how people are using this gem, what their pain points are, or if they aren't using the gem anymore what caused them to.

If you are interested in talking to email me at github@marktareshawty.com and we can work out a time, or feel free to comment on this issue!

Thanks!

**Comment by mafudge** [comment]
I plan on using the private repos / "Sandboxing" approach in my intro Java programming course. 

For teachers_pet, I had a rough time getting ruby 1.9.3 working on Windows 8.1 x64, more success with Ruby 2.1.5, FWIW.

I was getting certificate errors, which I fixed by following: https://gist.github.com/fnichol/867550 

Once I got through that problem, its been good. 

Any ideas / advice for how to give feedback to the student? I was planning on simply commenting their code, committing it as "graded" and pushing it back to their repository. That way they can see my comments highlighted in the diffs. Is there a better way?
